app/app/src/main/java/at/tugraz/ist/sw20/mam3/cook/ui/recipes/RecipesFragment.kt

```diff
@@ -1,25 +1,27 @@
 package at.tugraz.ist.sw20.mam3.cook.ui.recipes
 
 import android.os.Bundle
-import android.util.Log
+import android.view.ContextMenu
+import android.view.ContextMenu.ContextMenuInfo
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.AdapterView
+import android.widget.AdapterView.AdapterContextMenuInfo
 import android.widget.ListView
-import android.widget.TextView
 import android.widget.Toast
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.FragmentTransaction
-import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModelProvider
 import at.tugraz.ist.sw20.mam3.cook.R
-import at.tugraz.ist.sw20.mam3.cook.model.database.CookDB
 import at.tugraz.ist.sw20.mam3.cook.model.entities.Recipe
 import at.tugraz.ist.sw20.mam3.cook.model.service.DataReadyListener
 import at.tugraz.ist.sw20.mam3.cook.model.service.RecipeService
 import at.tugraz.ist.sw20.mam3.cook.ui.add_recipes.AddRecipesFragment
 import at.tugraz.ist.sw20.mam3.cook.ui.recipes.adapters.RecipeAdapter
 import com.google.android.material.floatingactionbutton.FloatingActionButton
+//import sun.jvm.hotspot.utilities.IntArray
+
 
 class RecipesFragment : Fragment() {
```

Comment:
Concerning the commit message: 
It is not in the correct form, it should be in the form COOK-# description of commit message [KA, --]
where the # would be the issue number, therefore in this branch it would be 'COOK-31' i think

Please try to fix this, if possible